cluster/os-3.11.0-multus/provider.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+source cluster/os-3.11.0/provider.sh
+
+image="os-3.11.0-multus@sha256:b7354263a7828cb13ae103f84c9d7c9d8def8f3edf6014a5e8e330724588afae"
```

Comment:
Can it be that this should be `os-3.11.0-multus@sha256:dfeaa7c1f7c264f953e44c93b50bb76c9bb58b0df8bb94dcaa108db254f87eb3`?

Reply:
I know but this is weird can you please try to run docker pull locally and tell me if it works for you? because it works for me.
`docker pull kubevirtci/os-3.11.0-multus@sha256:b7354263a7828cb13ae103f84c9d7c9d8def8f3edf6014a5e8e330724588afae`

Reply:
yes works for me. Maybe it was a transient issue? Which shasum is now the correct one?